packages/confirm-dialog/src/vaadin-confirm-dialog-mixin.js

```diff
@@ -450,11 +448,6 @@ export const ConfirmDialogMixin = (superClass) =>
       this.opened = false;
     }
 
-    /** @private */
-    _getAriaLabel(header) {
-      return header || 'confirmation';
```

Comment:
This was a leftover from Polymer version, it's not actually used after switching to Lit.